firmware/boron-ins-fsm/src/consoleFunctions.cpp

```diff
@@ -221,56 +221,63 @@ int ins_threshold_set(String input){
 
 }
 
-
-
-//particle console function to get/set door sensor ID
-int im21_door_id_set(String command) { // command is a long string with all the config values
+// particle console function to get/set door sensor ID
+// command is a long string with all the config values
+int im21_door_id_set(String command) { 
+  char buffer[64];
+  IMDoorID doorIDHolder; 
 
   if(isValidIM21Id(command) == false){
     return -1;
   }
 
-  //get pointer to user-entered string
+  // get pointer to user-entered string
   const char* checkForEcho = command.c_str();
 
-  //if echo, publish current door ID
-  if(*checkForEcho == 'e'){
-
-    IMDoorID holder;   
-    EEPROM.get(ADDR_IM_DOORID,holder.byte1);  
-    EEPROM.get((ADDR_IM_DOORID+1),holder.byte2);  
-    EEPROM.get((ADDR_IM_DOORID+2),holder.byte3);  
+  // if echo, publish current door ID
+  if(*checkForEcho == 'e'){  
+    EEPROM.get(ADDR_IM_DOORID, doorIDHolder.byte3);  
+    EEPROM.get((ADDR_IM_DOORID + 1), doorIDHolder.byte2);  
+    EEPROM.get((ADDR_IM_DOORID + 2), doorIDHolder.byte1);  
 
-    char buffer[64];
     snprintf(buffer, sizeof(buffer), "{\"byte1\":\"%02X\", \"byte2\":\"%02X\", \"byte3\":\"%02X\"}", 
```

Comment:
I've always found this output confusing. What do you think about changing the returned JSON to `{"doorId": "AB,CD,EF"}` where the string is in the same order as when it's given the the `im21_door_id_set` command?